repositories/src/test/scala/cromwell/pipeline/datastorage/dto/PipelineVersionTest.scala

```diff
@@ -0,0 +1,58 @@
+package cromwell.pipeline.datastorage.dto
+
+import cromwell.pipeline.datastorage.dto.PipelineVersion.PipelineVersionException
+import org.scalatest.{ AsyncWordSpec, Matchers }
+
+class PipelineVersionTest extends AsyncWordSpec with Matchers {
```

Comment:
```suggestion
class PipelineVersionTest extends WordSpec with Matchers {
```
Use `AsyncWordSpec` if you want to test Futures. Otherwise be more specific.

Reply:
Thanks for clarification! I'll fix it.